kurtosis-devnet/README.md

```diff
@@ -38,6 +38,11 @@ docker network rm kt-interop-devnet
 
 You can use `docker network ls` to inspect for networks to remove if the error message specifies some other network.
 
+### Existing enclave already running
+
+Set the `CLEAN_KT_DEVNET` envvar to `true` to make any `just devnet SPEC` based recipe stop and remove the existing
+enclave prior to attempting startup.
+
```

Comment:
Why wouldn't this just be the default given things don't work if there's an existing devnet?

Reply:
Things do work in some cases. You can re-run the NAT suite without re-starting the devnet, and that's usually preferred for the feedback loop to be shorter.

Reply:
Seems like there should be a way to just run tests rather than restart devnet and run tests then? I'm just trying to get to a point where `just interop-devnet` works because right now it fails far more often than it works.  If it's supposed to work even if an existing chain is running we should just fix that to be reliably. If it's meant to stop and rebuild the chain then we should make it always explicitly stop.  I'm not really sure what it's expected to do right now to be honest so not sure which way to go on that.

But to me having to set a magic env var is just another magic incantation people have to learn to make things work when it should just work in the first place.

Reply:
Flipped the logic around. Now `just *-devnet`'s default behavior is to attempt to clean an existing enclave, whereas `op-acceptance-tests`' usage of it skips the attempt to clean the devnet by default.

Should line up more with what folks are expecting there - faster test iteration with not tearing the whole devnet enclave down every time, and when directly using the devnet launcher, the old enclave is assumed to be trash (since a new devnet is explicitly being started.)